cypress-tests/cypress/e2e/configs/Payment/Commons.js

```diff
@@ -1569,4 +1569,57 @@ export const connectorDetails = {
       },
     },
   },
+  name_validation: {
+    HappyCase: getCustomExchange({
+      Request: {
+        currency: "USD",
+        billing: {
+          address: {
+            line1: "1467",
+            line2: "Harrison Street",
+            line3: "Harrison Street",
+            city: "San Fransico",
+            state: "California",
+            zip: "94122",
+            country: "US",
```

Comment:
I feel, most of this can be reused.

Please create an object at the top of the file. And then use spread operator to update the first and last name.